code/modules/mapping/random_rooms.dm

```diff
@@ -1479,3 +1479,139 @@
 	centerspawner = FALSE
 	template_height = 11
 	template_width = 5
+
+/datum/map_template/random_room/skwh_rdm158
+	name = "Large Abandoned Virology"
+	room_id = "skwh_rdm158_abandonedviro"
```

Comment:
Contains far too many disease samples AND Fungal TB.
TB is a nuke op only disease that is stronger than 90% of player created diseases.

Reply:
will revamp alot to be less too much of it as it is, plus while removing the Fungal TB, i should also remove the "WIZARD CONTAGION" bottle, that bottle is Wizarditis and i believe bottle might be a bit too much maybe?